test/fuzz/yaml/yaml.go

```diff
@@ -38,6 +79,25 @@ func FuzzSigYaml(b []byte) int {
 	return out
 }
 
+// FuzzTime is a fuzz target for unmarshaling Time defined in "k8s.io/apimachinery/pkg/apis/meta/v1".
+// This target also checks that the unmarshaled result can be marshaled back to the input.
+func FuzzTime(b []byte) int {
+	var unmarshalResult struct {
+		T metav1.Time `json:"t"`
+	}
+	if err := yaml.Unmarshal(b, &unmarshalResult); err != nil {
```

Comment:
i think that we are using `sigs.k8s.io/yaml` for objects like Time and Duration.
so would it make sense to have these new fuzzers use `sigs.k8s.io/yaml` instead of `yaml`?

Reply:
My bad, thanks for the catch!